### ISSUE TITLE
Separate in stock and out-of-stock products

### ISSUE THREAD

**Issue by briggySmalls** [briggySmalls/skin-deep-server#72]
The shop currently displays items that are both in-stock and out-of-stock. This is useful because in the future, digitised versions of articles might be uploaded and put behind a paywall. Then they would become 'in-stock' again, in digital form.

However, until that time the out-of-stock products are likely a lot less interesting to users. Should out-of-stock products be removed completely? Or is it desirable for users to be able to see previous editions (maybe this is a good showcase?).

I think the out of stock products should probably be separated from the others?

Note: This question is similar to the one about events #17

**Comment by CiaranChris** [comment]
I think it would be better to leave out of stock items in the shop for now.

When the user would click onto or hover over the item perhaps we could use a 'SOLD OUT badge' or image over top, and or on the description page of the item it should say 'SOLD OUT' and not give the option to buy until we've figured out the pay wall

**Comment by briggySmalls** [comment]
Yeah, so out-of-stock items cannot be purchased at the moment (the button is disabled). That is covered.

Out of interest, can I hear you motivation for keeping out of stock items in the shop. Is it to demonstrate previous editions as a showcase?

We can style sold out items any way you would like. As always, because of mobile, it probably would be nice to have a visual cue that doesn't require mouse over.

I was already wondering if there should be a more obvious 'call to action' on each product. Like a big green 'buy' button on the in-stock products. The out of stock products could have a greyed out one. The only issue then becomes will people know they can click on out of stock products to see a description?

**Comment by CiaranChris** [comment]
Reasoning for keeping the items in the shop was for if someone already had a subscription/log in once we've figured it out that perhaps they could use that as another way to navigate to past issue hosted behind the pay wall. 

Also to showcase what has been however thinking about it maybe it'd make more sense to only showcase the most current and what we actually have in-stock. 

I quite like what it's nice that has going on. They only show in-stock items but if you're browsing on the item page then you can see other items in stock and out of stock. https://www.companyofparrots.com/



I'm happy to redesign the buttons/ 'call to actions' as the fonts across the website will be changing to be bolder. I agree there that we have to make it very clear if we are going to be keeping in-stock and out of stock items in the shop.

**Comment by briggySmalls** [comment]
Sure. I get that sentiment. Let's talk about this tonight. I suggest we hide out of stock items until the paywall is live (at which point the item isn't really out of stock because it can be purchased online).

On 23 Oct 2018 13:07, CiaranChris <notifications@github.com> wrote:

Reasoning for keeping the items in the shop was for if someone already had a subscription/log in once we've figured it out that perhaps they could use that as another way to navigate to past issue hosted behind the pay wall.

Also to showcase what has been however thinking about it maybe it'd make more sense to only showcase the most current and what we actually have in-stock.

I quite like what it's nice that has going on. They only show in-stock items but if you're browsing on the item page then you can see other items in stock and out of stock. https://www.companyofparrots.com/

I'm happy to redesign the buttons/ 'call to actions' as the fonts across the website will be changing to be bolder. I agree there that we have to make it very clear if we are going to be keeping in-stock and out of stock items in the shop.

**Comment by briggySmalls** [comment]
Ok I understand now! Let's do it. Depends on #47

**Comment by briggySmalls** [comment]
For some reason jetpack's 'related posts' isn't showing anything on the products (despite showing for events). I'm not sure why (not enough of them maybe?)

**Comment by briggySmalls** [comment]
Hooray 🎉 we have related products now!

**Comment by briggySmalls** [comment]
Fixed
